app/models/user_track/external.rb

```diff
@@ -10,46 +10,29 @@ def initialize(track)
       @track = track
     end
 
-    delegate :concepts, :num_exercises, :num_concepts, :updated_at,
+    delegate :concepts, :num_exercises, :num_concepts, :updated_at, :course?,
       to: :track
     delegate :title, to: :track, prefix: true
 
     def user = nil
     def tutorial_exercise_completed? = false
     def anonymous_during_mentoring? = true
 
-    memoize
-    def exercises
-      enabled_exercises(track.exercises)
-    end
+    def solutions = Solution.none
```

Comment:
This is a trick to have the `scope :started, ->(user_track) { where(id: user_track.solutions.select(:exercise_id)) }` scope work for external user tracks too. It generates SQL that the query optimizer will detect as never giving any results and thus it will be skipped (`SELECT `solutions`.`exercise_id` FROM `solutions` WHERE (1=0))`).